cluster-up/cluster/kind-1.22-sriov/sriov-node/configure_vfs.sh

```diff
@@ -79,6 +79,9 @@ function ensure_driver_is_loaded() {
 
 DRIVER="${DRIVER:-vfio-pci}"
 DRIVER_KMODULE="${DRIVER_KMODULE:-vfio_pci}"
+VFS_COUNT=${VFS_COUNT:-0}
```

Comment:
We should default to the at least the previous behaviour but not to zero, defaults should be something that works.

Reply:
For what its worth, if VFS_COUNT is not overrides the script wont do any thing (also not mess anything).
I can set the default to 1, or 6 as we were going to use, WDYT?